pkg/optimizer/bad.fzn_solution

```diff
@@ -0,0 +1 @@
+ladder_num = array1d(1..14, [14, 6, 5, 13, 7, 8, 11, 9, 10, 1, 2, 3, 4, 12);
```

Comment:
I would suggest to move all test files (*.fzn_solution) to a subfolder `testdata`

Reply:
Done